### ISSUE TITLE
ossia.remote dumpout address index : wrong order

### ISSUE THREAD

**Issue by matcham** [ossia/libossia#733]
I'm trying to use this new feature : https://github.com/ossia/libossia/issues/657
(merci Vincent pour la bonne idée, et merci Julien de m'avoir rappelé que ça existe :-)

Looks like index does not always follow instance number.

if the multiple parameters are create by one single ossia.parameter : ossia.parameter joe.{1..3} 
then an ossia.remote joe.* will output index following the address
address: address joe.1 1
address: address joe.2 2
address: address joe.3 3

but if I create "manually" 3 parameters 
ossia.parameter bob.1
ossia.parameter bob.2
ossia.parameter bob.3

then ossia.remote bob.* will output 
address: address bob.3 1
address: address bob.2 2
address: address bob.1 3

<pre><code>
----------begin_max5_patcher----------
822.3oc0XssiaBCD84rR6+.B0mpnQ3w1ARep+GUUUlD2TVELQFy1Tsp+60XG
RX2RHlsNjceAT7ENyYlimYbd596lElUtmWEF74fuFLa1S5QlYFqYjYsCLKrf
se0VVkYggB9uJydHL5vbJ9dkY7cxbgJ3Q11Z9wIyWalRu9Os33fh5hbwVtx7
4PcFsrV0Nbb6v6XpU+LWr46R9Jk0PAzh4znfzz4wQAIjlm.LON3als7m6uq4
s9Uj6TptHiKCOgojUvUb424BV1V9yLnNThhFEmfA3DJNogGnTyKZ7IR0tI6G
R86cb6NBCiBByXhMgSOwIo9i3HvPbz6ChS8HwMbk.SJuypUpRwn4MBOJdiF5
zKAaB3lm.o84.79FEq8mFGSMzDQW9dPiCK7FwgDSlZvR425hbf5MQd5DKwK3
UUrM7+snLtuRwvYNMCimnDqxd4HBvWCZB8RSv2zDP2VZh5klHOSSxzDLGtOR
150RsanOBindqURbB0zDos4q3XOzK4YCdenWQJh3snWaYFbbSRWLbiheRMJ7
Aiei6boKUViMIbSW5XAlaPUUj+tq.Fa0oSbU0WIwS7W2DnaR2DmSmugok48V
ekNJ8M9BMKSaput7k20cX8cz04ncYUUNatjWTpodVY17O1G8IiJhiu7cErgZ
DN9MA8OJ7Md.bvWVy+AqdqpiVXatfuprVndVXuiKB6sCE1R1.hzoBFdw0MC3
HbOvI2C3t6A7r6AQnuMcOnStGj6tGjmcOIw+udG6FLV9K+2DMn1LwK7YUk0x
Use0CoMBPmPdMuRkKXpb8M4NsH8cSC5Zdx05RRMb+7gK+Z.zmuniFPrGLflF
i5xt9sflNFCFrCU2AD6BfdDOhK3k5M7ZtQ+kwq4BT8JpfoyDfqnt1QS.eFSH
1SGsfKJrgq7g6KaAzqnOf5jEf7l3m5R5LXgWw6hprDuAWpKvQ8G6PtfG1eot
vSKdDm7mf+3GLs3QbRd5uSe.50i2gtoX618HWVcXGVnzsZ9PoIkTZj824B6u
ss4EJ4Ol2tE60wCYRcaeJcOe0RiEFtewgqnEVTpSvIpyayx1P1FnMcyJzMpV
siYokoq26uSuf+RHfI5N
-----------end_max5_patcher-----------
</code></pre>

Maybe I don't understand what this index is supposed to represent..
index != instance ?
looks like index order depend of parameter creation order ; is that correct ?
So this is can be used only when using this kind of syntax : ossia.parameter joe.{1..3}
or maybe when parameters are created inside a poly (= only when you know for sure in which order parameters are created when the patch is opened. )

**Comment by avilleret** [comment]
> index != instance ?
right, index is always a number starting at 1, instance is alphanumerical

> looks like index order depend of parameter creation order ; is that correct ?
yes, it starts at one and increment each time you add a new node

> So this is can be used only when using this kind of syntax : ossia.parameter joe.{1..3}
another use is for poly~ instance dispatch and mc.* patchers

I don't know any other use case, but maybe you can explain what you're trying to do ?
